client/signup/style.scss

```diff
@@ -391,34 +395,63 @@ body.is-section-signup.is-white-signup .layout:not( .dops ):not( .is-wccom-oauth
 		font-weight: normal;
 	}
 
-	.formatted-header {
+	.step-wrapper__header {
 		margin-top: 48px;
 		margin-bottom: 40px;
 
+		@include breakpoint-deprecated( '<660px' ) {
+			margin-top: 24px;
+		}
+	}
+
+	.formatted-header {
+		margin: 0;
+
 		.formatted-header__title {
 			@include onboarding-font-recoleta;
 			color: $gray-100;
 			letter-spacing: 0.2px;
 			font-size: 2.25rem; /* stylelint-disable-line */
-			text-align: center;
-			padding: 0 20px;
 
 			@include break-xlarge {
 				font-size: 2.6rem; /* stylelint-disable-line */
-				padding: 0;
 			}
 		}
 
 		.formatted-header__subtitle {
 			color: $gray-60;
 			font-size: 1rem;
-			text-align: center;
+
+			@include breakpoint-deprecated( '<660px' ) {
+				margin-top: 16px;
+			}
+		}
+
+		.formatted-header__title,
+		.formatted-header__subtitle {
 			padding: 0 20px;
+			text-align: center;
 
 			@include break-xlarge {
 				padding: 0;
 			}
 		}
+
+		&.is-left-align {
+			.formatted-header__title,
+			.formatted-header__subtitle {
+				padding: 0;
```

Comment:
Just a nit, this rule could be up at line 434 and shared between the left and right styles

Reply:
How could I do it at line 434 🤔

Reply:
This is what I was thinking, maybe it doesn't work for some reason though

```scss
		.formatted-header__title,
		.formatted-header__subtitle {
			padding: 0 20px;
			padding: 0 20px;
			text-align: center;
			padding: 0; // Moved the padding=0 up here

			// ...
		}

		&.is-left-align {
			.formatted-header__title,
			.formatted-header__subtitle {
				// Removed the padding=0 here
				text-align: left;
			}
		}

		&.is-right-align {
			.formatted-header__title,
			.formatted-header__subtitle {
				// Removed the padding=0 here
				text-align: right;
			}
		}
```

Reply:
Oh, because I think we need to keep padding without left or right align

Reply:
Oh that makes sense, I forgot about centre alignment. This is good to go then.